src/Module/Login.php

```diff
@@ -347,4 +378,56 @@ public static function form($return_path = null, $register = false, $hiddens = [
 
 		return $o;
 	}
+
+	/**
+	 * Get the URL to the register page and add OpenID parameters to it
+	 */
+	private static function getRegisterURL()
+	{
+		if (empty(Session::get('openid_identity'))) {
+			return 'register';
+		}
+
+		$args = [];
+		$attr = Session::get('openid_attributes') ?? [];
```

Comment:
`Session::get` actually has a second parameter to indicate the default value if none exists for the provided key. I'm not sure which way is more readable, it's up to you.
```suggestion
		$attr = Session::get('openid_attributes', []);
```